frontend/src/constants/sidebarOptions.ts

```diff
@@ -4,6 +4,13 @@ const sidebarOptions = [
   { name: 'Employees', endpoint: 'employees' },
 ];
 
+export const SIDEBAR_STATES = {
+  CLOSED: 'closed',
+  CLOSING: 'closing',
+  OPEN: 'open',
+  OPENING: 'opening',
+};
+
```

Comment:
Is this used anywhere?

Reply:
ah used it in a previous iteration and forgot to remove